### PR TITLE
Migrate from Goerli to Holesky

### PR DESCRIPTION
**Pull request by mateuszradomski**
Resolves L2B-2668

### CODE REVIEW

File: packages/node/src/core/BatchDownloader.ts
Comment (lucadonnoh) [inline packages/node/src/core/BatchDownloader.ts:1]: what's the reason behind these changes?

Comment (mateuszradomski) [reply packages/node/src/core/BatchDownloader.ts:1]: The version of viem had to be upgraded to include the Holesky testnet. The new version changed some of the fields from being _optional_ to **required** and vice-versa. These changes are just making sure that type system is happy and the linter does not throw any errors.

**Review by lucadonnoh — APPROVED** [review]
